kafka-connect-mqtt/src/main/scala/com/datamountaineer/streamreactor/connect/mqtt/config/MqttSourceSettings.scala

```diff
@@ -120,7 +121,8 @@ object MqttSourceSettings {
       sslCACertFile,
       sslCertFile,
       sslCertKeyFile,
-      progressEnabled
+      progressEnabled,
```

Comment:
my comment is not for this change but in generally why are we using so big constructors and not builders for examples? One of the best practices and by Joshua if I remember correctly was:  "Consider a
builder when you have many constructor parameters"